<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dependece</class>
 <widget class="QDialog" name="dependece">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Dependencias</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fazendo dolwnload das dependências:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="logText"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_install_dep">
     <property name="text">
      <string>Clique aqui para iniciar o download das dependências</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="concluir">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Concluir</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
